examples/gpu_cache.rs

```diff
@@ -1,242 +1,261 @@
+#[cfg(feature = "gpu_cache")]
 #[macro_use]
 extern crate glium;
+#[cfg(feature = "gpu_cache")]
 extern crate rusttype;
+#[cfg(feature = "gpu_cache")]
 extern crate arrayvec;
+#[cfg(feature = "gpu_cache")]
```

Comment:
You might be able to remove the need for these individual `cfg`s by moving each `extern crate *` into the feature gated `gpu_cache` module?